crates/sui-storage/src/indexes.rs

```diff
@@ -419,6 +424,15 @@ impl IndexStore {
             }),
         )?;
 
+        // Loaded child objects table
+        if let Some(loaded_child_objects) = loaded_child_objects {
+            let loaded_child_objects: Vec<_> = loaded_child_objects.into_iter().collect();
```

Comment:
Why was this an Option? It will always be `Some` if we ever enter execution right? 
Maybe just skip writing it if the map is empty? I feel like that would have similar results unless I'm missing something